helpers/partial.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
 const factory = globals => {
-    return function(name) {
-        const options = arguments[arguments.length - 1];
+    return function(name, options) {
```

Comment:
Perhaps the safer thing is

```
function(...args) {
  const options = args.pop();
  const name = args[0];
}
```

Then the tests won't have to change, perhaps that test was added for backward compatibility with an unused parameter.

Reply:
fixed for partial.js, block.js and stylesheet.js set test cases back to normal.